proprietary/Components/src/denhaag/card-calendar.tokens.json

```diff
@@ -0,0 +1,164 @@
+{
+  "denhaag": {
+    "card-calendar": {
+      "wrapper": {
+        "display": { "value": "grid" },
+        "grid-template-columns": { "value": "repeat(12, 1fr)" },
+        "xs": {
+          "column-gap": { "value": "0" },
+          "row-gap": { "value": "55px" },
+          "grid-column": { "value": "span 12" }
+        },
+        "sm": {
+          "gap": { "value": "16px" },
```

Comment:
Hiervoor gebruiken wij de space.tokens.json.
In dit geval kun je dit doen `"gap": { "value": "{denhaag.space.block.md}" }`